frontend_lib/src/component/TinyEditor/TinyEditor.jsx

```diff
@@ -24,7 +24,10 @@ const advancedToolBar = [
   'code codesample | insert | removeformat | customFullscreen help'
 ].join('')
 
-const simpleToolBar = 'bold italic underline | bullist numlist | customFullscreen help'
+const simpleToolBar = [
+  'bold italic underline | bullist numlist | link customInsertImage emoticons | ',
+  'customFullscreen help'
+].join('')
```

Comment:
I guess this is to add emojis and links in the standard view, might be good to explicit it in the PR title or desc

Reply:
True